constants/limit.go

```diff
@@ -24,6 +24,9 @@ const (
 	// BuildTimeoutDefault defines the default value in minutes for repo build timeout.
 	BuildTimeoutDefault = 30
 
+	// ErrorLimit defines the maximum size in characters for resource error fields.
```

Comment:
Oh, one small thing, this comment is helpful, but it doesn't let anyone know what happens if the limit is exceeded. They'd have to dig into the code to understand that half of the beginning and half of the end are kept. It might be beneficial for future folks looking in the codebase to understand the intent of that and what happens if it gets exceeded.